### PR TITLE
Set squiggle bonds to STEREOANY with stereo atoms

### LINKED ISSUE #6876: Wavy bonds in mol blocks can't be stereo enumerated
Test case.

If we round trip them through cx smiles, they can be stereo enumerated.

```
  SECTION("ensure we can enumerate stereo on either double bonds") {
    auto mol = R"CTAB(
  Mrv2004 11072316002D          

  0  0  0     0  0            999 V3000
M  V30 BEGIN CTAB
M  V30 COUNTS 8 8 0 0 1
M  V30 BEGIN ATOM
M  V30 1 C 1.859 2.7821 0 0
M  V30 2 C 3.2818 2.1928 0 0
M  V30 3 C 3.8711 0.77 0 0
M  V30 4 C 3.2818 -0.6528 0 0
M  V30 5 C 1.859 -1.2421 0 0
M  V30 6 C 0.4362 -0.6528 0 0
M  V30 7 C -0.1533 0.77 0 0
M  V30 8 C 0.4362 2.1928 0 0
M  V30 END ATOM
M  V30 BEGIN BOND
M  V30 1 1 1 2
M  V30 2 1 2 3
M  V30 3 1 4 3 CFG=2
M  V30 4 2 4 5
M  V30 5 1 5 6
M  V30 6 1 6 7
M  V30 7 1 7 8
M  V30 8 1 1 8
M  V30 END BOND
M  V30 END CTAB
M  END)CTAB"_ctab;
      //mol->debugMol(std::cerr);
      std::string smi = MolToCXSmiles(*mol, SmilesWriteParams());
      std::cout << smi << std::endl;
      auto f = SmilesToMol(smi);
      f->getBondWithIdx(0)->setStereo(Bond::BondStereo::STEREOCIS);
      mol->getBondWithIdx(3)->setStereo(Bond::BondStereo::STEREOCIS);
      delete f;
  }
}
```

### PR DESCRIPTION
**Pull request by bp-kelley**
Fixes #6876 

This allows squiggle bonds in mol blocks to be stereo enumerated.  If we round trip through CXSmiles, the bonds are stereo enumerable so this is intended behavior.

### CODE REVIEW

**Comment by bp-kelley** [comment]
@greglandrum it will be interesting to see what this breaks :)

File: Code/GraphMol/Chirality.cpp [hunk 1]
```diff
@@ -153,6 +153,46 @@ void controllingBondFromAtom(const ROMol &mol,
   }
 }
 
+namespace {
+void set_stereoany_from_squigglebond(ROMol &mol, Bond *bond) {
```
Comment (greglandrum) [inline Code/GraphMol/Chirality.cpp:157]: a trivial comment before I do a real review (that's for tomorrow): we don't use snake_case in the RDKit. Please switch this to
```suggestion
void setStereoAnyFromSquiggleBond(ROMol &mol, Bond *bond) {
```

Comment (bp-kelley) [reply Code/GraphMol/Chirality.cpp]: I moved this into the detail namespace and also used it in CXSmilesOps to remove duplicated code.

**Comment by bp-kelley** [comment]
@greglandrum  It's compiling again.

**Comment by bp-kelley** [comment]
@greglandrum gentle poke on this one.

**Review by greglandrum — APPROVED** [review]
LGTM

**Pull request merged**